src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -297,13 +297,15 @@ else if (value.startsWith("(default)")) {
     }
 
     private static void setViolations(TestInputConfiguration.Builder inputConfigBuilder,
-                                      List<String> lines, boolean useFilteredViolations)
+                                      List<String> lines, boolean useFilteredViolations,
+                                      String inputFilePath)
             throws ClassNotFoundException, CheckstyleException {
         final List<ModuleInputConfiguration> moduleLists = inputConfigBuilder.getChildrenModules();
         final boolean specifyViolationMessage = moduleLists.size() == 1
                 && !SUPPRESSED_CHECKS.contains(moduleLists.get(0).getModuleName())
                 && !PERMANENT_SUPPRESSED_CHECKS.contains(moduleLists.get(0).getModuleName())
-                && getNumberOfMessages(moduleLists.get(0).getModuleName()) > 1;
+                && getNumberOfMessages(moduleLists.get(0).getModuleName()) > 1
+                && !inputFilePath.contains("xdocs-test");
```

Comment:
We try to keep examples short so we do not require violation messages in them.

Reply:
We can move forward with messages and in few Checks how it looks and remove them if it looks bad.

Or we can move forward without messages, and at the middle of process estimate to add messages or not.

Think on this, and let us know.

Reply:
I'd rather go with messages and remove them later if necessary. Even better, we can make examples input parser to omit those messages if we want to.

Reply:
Looks good for now with messages, but we will see on complicated case how it looks in next PRs